client/src/components/ListingItem.jsx

```diff
@@ -109,11 +109,11 @@ const ListingItem = ({ content, id}) => {
         </Grid>
       );
     }
-  }
+  };
 
   if (content.contentType === 'component') {
```

Comment:
Can you instead have default objects and then extend the different keys. This would remove a lot of this duplication